template/.npmignore.temp

```diff
@@ -1,16 +1,23 @@
 # standard ignore
+*config*
+*conf*
+
+.github/
 .gitignore
+.eslintrc
 .stylelintrc
-karma.conf.js
-rollup.config.js
-babel.config.js
-.github/
-index.html
+
+coverage/
+docs/
 scripts/
 test/
-demo/css
-demo/sass
-coverage/
+demo/*.css
+demo/*.scss
+demo/*.html
+
+dist/es5.js
```

Comment:
Whats the reason to stop including the es5 bundle with the npm install process?

Reply:
The es5.js is not a bundle. It's a dependency in the bundling process, but not needed in the final bundle.